contracts/interfaces/IWildCardSteward.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.5.1;
+
+import "@nomiclabs/buidler/console.sol";
+
+interface IWildCardSteward
+{
```

Comment:
Here from line 32
`mapping(uint256 => address) public currentPatron; // This is different to the current token owner.`
We need to check who the owner of the dragonToken is from this mapping before calling the collectPatronage :) 
So I think we maybe need to add this value to the interface :)

Reply:
done :)